specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -7688,130 +8132,492 @@
+          "minimum": 0,
+          "description": "An integer representing the interval in minutes between two backups"
+        },
+        "backupRetentionIntervalInHours": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "An integer representing the time (in hours) that each backup is retained"
+        }
+      }
+    },
+    "RestorableDatabaseAccountsListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RestorableDatabaseAccountGetResult"
+          },
+          "description": "List of restorable database accounts and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the restorable database accounts and their properties."
+    },
+    "RestorableDatabaseAccountGetResult": {
+      "description": "A Azure Cosmos DB restorable database account.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a restorable database account.",
+          "$ref": "#/definitions/RestorableDatabaseAccountProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ARMResourceProperties"
+        }
+      ]
+    },
+    "RestorableDatabaseAccountProperties": {
+      "type": "object",
+      "description": "The properties of a restorable database account.",
+      "properties": {
+        "accountName": {
+          "type": "string",
+          "description": "The name of the global database account"
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The creation time of the restorable database account (ISO-8601 format)."
+        },
+        "deletionTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time at which the restorable database account has been deleted (ISO-8601 format)."
+        },
+        "apiType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Enum to indicate the API type of the restorable database account.",
+          "$ref": "#/definitions/ApiType"
+        },
+        "restorableLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of regions where the of the database account can be restored from.",
+          "items": {
+            "$ref": "#/definitions/RestorableLocationResource"
+          }
+        }
+      }
+    },
+    "ApiType": {
+      "description": "Enum to indicate the API type of the restorable database account.",
+      "type": "string",
+      "readOnly": true,
+      "enum": [
+        "MongoDB",
+        "Gremlin",
+        "Cassandra",
+        "Table",
+        "Sql",
+        "Etcd",
```

Comment:
In my knowledge, we dont support it anymore. Let us refrain from exposing it here.